src/components/Menu/Menu.js

```diff
@@ -61,7 +61,7 @@ class Menu extends React.Component {
     return (
       <div style={currentStyle.bar} >
         <div style={{flex: 1}}>
-          <Link to='/' style={{textDecoration: 'none'}}>
+          <Link to={basePath ? basePath : '/'} style={{textDecoration: 'none'}}>
```

Comment:
Maybe it would be better to always set `basePath` during configuration and make it required everywhere else.